test_helpers/mocks/wrappers/core/state/database.go

```diff
@@ -11,39 +11,31 @@ import (
 )
 
 type MockStateDatabase struct {
-	returnDB state.Database
-	trie     state_wrapper.GethTrie
+	ReturnDB   state.Database
+	ReturnTrie state_wrapper.GethTrie
 }
 
 func NewMockStateDatabase() *MockStateDatabase {
 	return &MockStateDatabase{
-		returnDB: nil,
-		trie:     nil,
+		ReturnDB:   nil,
+		ReturnTrie: nil,
```

Comment:
This is my bad from a long time ago but we probably don't need to specify these values in the initializer if they're just being assigned as `nil`